specs/protocol/overview.md

```diff
@@ -142,12 +143,12 @@ graph LR
 
 #### Notes for Core L1 Smart Contracts
 
-- The `Batch Inbox Address` described above (**highlighted in GREY**) is *not* a smart contract and is instead an arbitrarily
+- The `Batch Inbox Address` described above (**highlighted in GREY**) is _not_ a smart contract and is instead an arbitrarily
```

Comment:
Just noting this, when OPCM launches then it is no longer an arbitrarily selected account, it has a definition